modules/ROOT/pages/datagraph-landing-page.adoc

```diff
@@ -1,16 +1,20 @@
-= DataGraph Administration
+= Anypoint DataGraph Administration
```

Comment:
Should we call this Configure DataGraph or something like that?

Reply:
Let me run this by Kim. It's not really "configuring"